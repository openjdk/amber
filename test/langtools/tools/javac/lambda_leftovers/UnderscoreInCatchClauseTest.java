/*
 * @test /nodynamiccopyright/
 * @bug 8173059
 * @summary Underscore for unnamed method, constructor, lambda, and catch formals
 * @compile/fail/ref=UnderscoreInCatchClauseTest.out -XDrawDiagnostics UnderscoreInCatchClauseTest.java
 */

public class UnderscoreInCatchClauseTest {
    void foo() {
        // this use should be allowed
        try { } catch (Throwable _) {
            // this use should be forbidden
            throw _;
        }
    }
}
