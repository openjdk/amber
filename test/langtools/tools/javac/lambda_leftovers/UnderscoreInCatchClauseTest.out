UnderscoreInCatchClauseTest.java:13:19: compiler.err.underscore.not.allowed
1 error
